config/chef/data_bags/users/deploy.json.example

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "deploy",
+  "comment": "Deploy user",
+  "shell": "/bin/bash",
+
+  // TODO: Include instructions on encrypting a password
+  "password": "REPLACE ME",
+
+  "ssh_keys": [
+    "REPLACE ME"
+  ]
+}
```

Comment:
That's corrected in #1029 [here](https://github.com/opengovfoundation/madison/blob/6666f780d3c96ec0b02d08f23aeedc34a5f38a95/config/chef/data_bags/users/you.json.example)